recipe/meta.yaml

```
@@ -24,18 +24,18 @@ requirements:
     - python {{ python_min }}
     - pip
     - babel
-    - flit-core >=3.11
+    - flit-core >=3.12
   run:
     - python >={{ python_min }}
     - alabaster >=0.7.14
     - babel >=2.13
-    - docutils >=0.20,<0.22
+    - docutils >=0.20,<0.23
     - imagesize >=1.3
     - jinja2 >=3.1
     - packaging >=23.0
     - pygments >=2.17
     - requests >=2.30.0
-    - roman-numerals-py >=1.0.0
+    - roman-numerals >=1.0.0
```

There is no `roman-numerals` feedstock at present.
```suggestion
    - roman-numerals-py >=1.0.0
```